packages/server/src/constants/pages.ts

```diff
@@ -1,4 +1,4 @@
 export enum Pages {
   EMAIL_CONFIRMATION_PAGE = '/confirm',
-  PASSWORD_RESET_PAGE = '/password-reset'
+  PASSWORD_RESET_PAGE = '/reset'
```

Comment:
There is a difference between your changes and Oleksiy's one in feat(12) packages/web/src/App.tsx
I'd suggest use one path and page name

Reply:
I prefer this one, but please update the URL used in the emails